swift/ql/src/queries/Security/CWE-020/IncompleteHostnameRegex.ql

```diff
@@ -0,0 +1,16 @@
+/**
+ * @name Incomplete regular expression for hostnames
+ * @description Matching a URL or hostname against a regular expression that contains an unescaped dot as part of the hostname might match more hostnames than expected.
```

Comment:
The description looks fine, but perhaps we could replace the second occurrence of "to match" so that there's less repetition. I've used "to return" but perhaps you can find a better synonym. Also used the present for "may".
```suggestion
 * @description Matching a URL or hostname against a regular expression that contains an unescaped dot as part of the hostname may return more hostnames than expected.
```

Reply:
:+1: "may", I've made that change.

I'm not quite happy with the word "return" here since we don't know whether the user code we've identified is really "returning" anything (or whether it's doing something else with the match).  Sadly I can't find a good alternative word so I've left the repetition of "match" for now.

Reply:
@geoffw0 - that's fine, let's lave it as-is. Thanks for addressing my other comments 👍🏻